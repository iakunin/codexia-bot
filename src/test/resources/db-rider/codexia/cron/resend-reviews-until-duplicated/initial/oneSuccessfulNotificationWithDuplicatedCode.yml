codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        project_created_at: "2019-12-20 12:01:02"

codexia_review:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 5
        author: Review author
        reason: Review reason
        text: Review text

codexia_review_notification:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_review_id: 1
        status: SUCCESS
        response_code: 404
        response: Some response
