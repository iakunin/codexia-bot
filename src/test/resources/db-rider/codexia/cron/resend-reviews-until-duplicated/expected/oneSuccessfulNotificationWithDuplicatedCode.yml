codexia_review_notification:
    -
        codexia_review_id: 1
        status: SUCCESS
        response_code: 404
        response: Some response
